Does pd address has to take an http prefix?

If I don't add an http prefix, pd-ctl will panic like:

```
address is wrong format,should like 'http://127.0.0.1:2379'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x73cc0c]

goroutine 1 [running]:
github.com/pingcap/pd/pdctl/command.getAddressFromCmd(0xc4203b6fc0, 0xca2f37, 0x10, 0xc420285938, 0x70243f)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/pdctl/command/global.go:113 +0x9c
github.com/pingcap/pd/pdctl/command.getRequest(0xc4203b6fc0, 0xca2f37, 0x10, 0xc9a128, 0x4, 0xca2b17, 0x10, 0xd3aba0, 0xc420297a40, 0x0, ...)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/pdctl/command/global.go:61 +0x73
github.com/pingcap/pd/pdctl/command.postConfigDataWithPath(0xc4203b6fc0, 0xc42025a72b, 0x13, 0xc42025a73f, 0x3, 0xca2f37, 0x10, 0xc4203b8780, 0xc9a3a0)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/pdctl/command/config_command.go:223 +0x208
github.com/pingcap/pd/pdctl/command.setConfigCommandFunc(0xc4203b6fc0, 0xc42028af00, 0x2, 0xa)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/pdctl/command/config_command.go:240 +0x95
github.com/pingcap/pd/vendor/github.com/spf13/cobra.(*Command).execute(0xc4203b6fc0, 0xc42028adc0, 0xa, 0xa, 0xc4203b6fc0, 0xc42028adc0)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/vendor/github.com/spf13/cobra/command.go:702 +0x2c6
github.com/pingcap/pd/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4203b6000, 0xe, 0xc4203c2000, 0xc4203b6140)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/vendor/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/pingcap/pd/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4203b6000, 0xc4203b4000, 0xc)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/vendor/github.com/spf13/cobra/command.go:736 +0x2b
github.com/pingcap/pd/pdctl.Start(0xc4203b4000, 0xc, 0x10)
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/pdctl/ctl.go:79 +0x752
main.loop()
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/cmd/pd-ctl/main.go:127 +0x3c4
main.main()
	/home/jenkins/workspace/build_pd_master/go/src/github.com/pingcap/pd/cmd/pd-ctl/main.go:93 +0x361
```

It will be more friendly to add an http prefix for address that doesn't have one automatically. And the program should not panic with segmentation violation too.

--- a/cmd/pd-ctl/main.go
+++ b/cmd/pd-ctl/main.go
@@ -125,6 +125,9 @@ func loop() {
 			fmt.Printf("parse command err: %v\n", err)
 			continue
 		}
+		if url != "" && !strings.HasPrefix(url, "http") {
+			url = "http://" + url
+		}
 		args = append(args, "-u", url)
 		if caPath != "" && certPath != "" && keyPath != "" {
 			args = append(args, "--cacert", caPath, "--cert", certPath, "--key", keyPath)

